### ISSUE TITLE
Improving `sum_of_powers` snippet

### ISSUE THREAD

**Issue by gurukiran07** [30-seconds/30-seconds-of-python#354]
Current implementation
```python
def sum_of_powers(end, power = 2, start = 1):
  return sum([(start + i) ** power for i in range(0, end + 1 - start)])
```

Can be refactored into
```python
def sum_of_powers(end, power = 2, start = 1):
  return sum([(i) ** power for i in range(start, end + 1)])
```
Produces the same results.

**Comment by Trinityyi** [comment]
Sure, you can go ahead and PR this. Also, please add `Resolves #354` in the PR to auto-close the issue when merged.

**Comment by gurukiran07** [comment]
@Trinityyi Submitted a PR #357 .
